Jenkinsfile

```diff
@@ -14,12 +14,6 @@ pipeline {
         }
 
         stage("Clean") {
-            when {
```

Comment:
I removed this because it gave a wrong or incomplete rest result recording for branches.